source/ui/dataman/AM2DScanView.cpp

```diff
@@ -492,7 +492,52 @@ void AM2DScanView::mousePressEvent(QMouseEvent *e)
 
 void AM2DScanView::setPlotRange(double low, double high)
 {
-	spectrumView_->setPlotRange(low, high);
```

Comment:
Since the scan view methods just call this renamed function, should the scan view methods be renamed as well?